src/modules/commander/state_machine_helper.cpp

```diff
@@ -1039,15 +1051,22 @@ void battery_failsafe(orb_advert_t *mavlink_log_pub, const vehicle_status_s &sta
 			break;
 
 		case LOW_BAT_ACTION::RETURN:
-			if (status_flags.condition_global_position_valid && status_flags.condition_home_position_valid) {
+			if (internal_state->main_state == commander_state_s::MAIN_STATE_AUTO_RTL) {
+				mavlink_log_critical(mavlink_log_pub, "%s, already in RTL", battery_dangerous);
+
+			} else if (internal_state->main_state == commander_state_s::MAIN_STATE_AUTO_LAND ||
+				   internal_state->main_state == commander_state_s::MAIN_STATE_AUTO_PRECLAND) {
+				mavlink_log_critical(mavlink_log_pub, "%s, already landing", battery_dangerous);
```

Comment:
```suggestion
				mavlink_log_critical(mavlink_log_pub, "%s, landing", battery_dangerous);
```

Reply:
Changed to 'continuing landing', I think it better.